docs/guides/context.md

```diff
@@ -323,17 +323,22 @@ const countMachine = createMachine({
 
 ## TypeScript
 
-For proper type inference, add the context type as the first type parameter to `createMachine<TContext, ...>`:
+For proper type inference, add the context type to the schema property of the machine:
```

Comment:
Maybe we should emphasize somewhere here that using explicit generics is not recommended? We could label that note with the version of XState that will include the typegen support.

Reply:
@Andarist This is covered by the VSCode extension itself. It gives you an error line if you use explicit generics. Should we also add a note here?

Reply:
I guess it's OK as it is - let's ride on and observe what happens 😉